### PR TITLE
Add TeamViewer Script and Integration Docs

### PR DESCRIPTION
**Pull request by meuchels**
Thanks for the help silversword.

### CODE REVIEW

**Comment by silversword411** [comment]
Looks good!

**Comment by meuchels** [comment]
I have added an AnyDesk one as well.

File: api/tacticalrmm/scripts/community_scripts.json [hunk 1]
```diff
@@ -180,6 +180,30 @@
...
+    "category": "TRMM (Win):Collectors"
+  },
+  {
+    "guid": "e43081d4-6f71-4ce3-881a-22da749f7a57",
+    "filename": "Win_AnyDesk_Get_Anynet_ID.ps1",
+    "submittedBy": "https://github.com/meuchels",
+    "name": "AnyDesk - Get AnyNetID for client",
+    "description": "Returns AnyNetID for client - Use with Custom Fields for later use. ",
+    "args": [
+      "-serviceName {{client.ScreenConnectService}}"
```
Comment (silversword411) [inline api/tacticalrmm/scripts/community_scripts.json:202]: This doesn't match Anydesk, is this even needed? I'll have to think about it

File: api/tacticalrmm/scripts/community_scripts.json [hunk 2]
```diff
@@ -180,6 +180,30 @@
...
     "category": "TRMM (Win):Collectors"
   },
+  {
+    "guid": "9cfdfe8f-82bf-4081-a59f-576d694f4649",
+    "filename": "Win_Teamviewer_Get_ID.ps1",
+    "submittedBy": "https://github.com/silversword411",
+    "name": "TeamViewer - Get ClientID for client",
+    "description": "Returns Teamviwer ClientID for client - Use with Custom Fields for later use. ",
+    "args": [
+      "-serviceName {{client.ScreenConnectService}}"
```
Comment (silversword411) [inline api/tacticalrmm/scripts/community_scripts.json:190]: Doesn't match teamviewer, is this needed?

File: api/tacticalrmm/scripts/community_scripts.json (hunk 1 above)
Comment (meuchels) [reply api/tacticalrmm/scripts/community_scripts.json:202]: I did not notice args portion. It is not needed in TeamViewer or Anydesk

**Pull request merged**